package.json

```diff
@@ -9,11 +9,21 @@
   ],
   "license": "Apache-2.0",
   "main": "./dist/lib/index.js",
+  "files": [
```

Comment:
Why would the submodules be installed as dependencies?

Reply:
Because if you don't get a pre-built binary, you need them to build. Getting them could probably be moved to the build step at some point, but it would be tricky.

Reply:
Ok, I guess we'll go with it. It would be idea to avoid needing them eventually.